### PR TITLE
David/sanity migration

### PR DESCRIPTION
**Pull request by haveanicedavid**
Sanity changes for regen-network/regen-web#797

### CODE REVIEW

File: deskStructure.js [hunk 1]
```diff
@@ -16,154 +16,302 @@ import S from '@sanity/desk-tool/structure-builder';
...
-        .title('Website'),
+      // Website
+      S.listItem()
+        .title('Website')
+        .child(
+          S.list()
+            .title('Website')
+            .items([
+              S.listItem()
+                .title('Home Web')
```
Comment (blushi) [inline deskStructure.js]:
```suggestion
                .title('Home Page')
```

File: schemas/documents/www/homePageWeb.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+export default {
+  name: 'homePageWeb',
+  type: 'document',
+  title: 'Web Home Page',
+  __experimental_actions: ['update', 'create', /*'delete', */ 'publish'],
```
Comment (blushi) [inline schemas/documents/www/homePageWeb.js]:
```suggestion
  __experimental_actions: ['update', /*'create', 'delete', */ 'publish'],
```
ditto for all pages

File: .prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (blushi) [inline .prettierrc.json:1]: +1 for adding prettier config
but it seems like not all files are properly formatted
should we add prettier to the dev dependencies too?

File: schemas/documents/www/homePageWeb.js (hunk 2 above)
Comment (haveanicedavid) [reply schemas/documents/www/homePageWeb.js]: @blushi FYI it looks like they're [deprecating this feature](https://www.sanity.io/docs/ui-affordances-for-actions)

I created a [new issue](https://app.zenhub.com/workspaces/regen-registry-5f8998bec8958d000f4609e2/issues/regen-network/regen-registry/787) to track so we can go ahead and get the sanity stuff merged first

File: .prettierrc.json (hunk 3 above)
Comment (haveanicedavid) [reply .prettierrc.json:1]: Just pushed an update doing just that + an npm script to run across the repo

**Pull request merged**